servers/cu/src/effects/worker/evaluator/index.js

```diff
@@ -9,6 +9,55 @@ import { arrayBufferFromMaybeView } from '../../../domain/utils.js'
 
 import { createApis } from './main.js'
 
+class EventVacuum {
+  constructor (transport) {
+    this.transport = transport
+  }
+
+  // Method to process log lines and filter events
+  processLogs (logData, processId, nonce) {
+    const events = (logData ?? '')
+      .split('\n')
+      .map(this.parseJson)
```

Comment:
this would preclude multiline json strings no?

Reply:
Correct. The example code I've created for the event printer ensures that the events are printed in a single line. Newline characters are allowed (i.e. ndjson).